docs/unified-test-documentation/generic-testing-stand-setup.md

```diff
@@ -0,0 +1,351 @@
+[here](https://docs.dasharo.com/transparent-validation/pikvm/assembly-and-validation/).
+
+#### Platform external flashing
+
+Platform external flashing is needed for two reasons:
+
+* it enables quick changes to the firmware,
+* it enables the process of unbricking the platform.
+
+The flashing operation usually consists of several commands, involving crucial
+power management, so it is advisable to prepare a script that automates the
+process. This is a crucial function, so if in doubt, ask your supervisor.
+
+> NOTE: On some platforms, the flash chip supply voltage is 1.8V, not 3.3V.
+  The supply voltage should be determined before making any connections, based
+  on our platform-specific documentation or the flash chip datasheet.
+
+Example script template:
+
+```bash
+#!/bin/bash
+
+# Platform power control, example commands:
+./sonoff.sh off
+rte_ctrl poff
+echo 0 > /sys/class/gpio/gpio199/value
+
+# Set appropriate GPIO states
+echo "set SPI Vcc to 3.3V"
+echo 1 > /sys/class/gpio/gpio405/value
+sleep 2
+echo  "SPI Vcc ON"
+echo 1 > /sys/class/gpio/gpio406/value
+sleep 2
+echo "SPI lines ON"
+echo 1 > /sys/class/gpio/gpio404/value
+sleep 2
+
+# dry flash command
+echo "start flash command"
+flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000
+
+# Set back GPIO states
+echo "Turn SPI off"
+echo 0 > /sys/class/gpio/gpio404/value
+echo 0 > /sys/class/gpio/gpio406/value
+
+
+# Platform power control, example commands:
+./sonoff.sh on
+echo 1 > /sys/class/gpio/gpio199/value
+rte_ctrl poff
+```
+
+The output should resemble the following:
+
+```bash
+(...)
+flashrom v1.2 on Linux 5.4.69 (armv7l)
+flashrom is free software, get the source code at https://flashrom.org
+
+Using clock_gettime for delay loops (clk_id: 1, resolution: 1ns).
+Found Macronix flash chip "MX25U25635F" (32768 kB, SPI) on linux_spi.
+(...)
+```
+
+> Flashing with flashrom takes about 1 minute.
+
+Note that the first boot of the platform after proceeding with above procedure
+can take much longer than usual.
+
+### Stand Setup
+
+Before placing the stand in the lab, it is recommended to set it up first at
+your desk to verify its functionality. After compiling the stand, the
+connections that we have made should be checked. The necessary thing to do the
+check is the IP address of RTE which we can get by connecting to RTE via serial
+using minicom and using the `ip a` command.
+
+After this, we have an IP to connect via ssh. This process should suit
+every position:
+
+1. Connect to RTE via ssh.
+1. Check access to the DUT using the `telnet <IP> <port>` command from RTE or
+   log in to PiKVM by entering its IP address in your browser.
+1. Check that the commands responsible for the power control are working
+   properly:
+
+    ```sh
+    rte_ctrl rel
+    ```
+
+    ```sh
+    rte_ctrl pon
+    ```
+
+    ```sh
+    rte_ctrl poff
+    ```
+
+    ```sh
+    rte_ctrl reset
+    ```
+
+    > NOTE: On different versions of the RTE software, the invocation of the
```

Comment:
If we use `osfv_cli`, the version of RTE image does not matter that much, if any,

Reply:
Removed the note and [replaced](https://github.com/Dasharo/docs/pull/753/commits/3b1e9b8738ab810b772a98c7100876b148565496#diff-21b728890d92d73a2d9a1746109ea2d9309b6b02efbcf910fbe8ffc652a6c269R288) all `rte_ctrl` commands with `osfv_cli` equivalents